### PR TITLE
Unauthenticated share updates adjustments

### PR DESCRIPTION
**Pull request by timarney**
# Summary | Résumé

Adjustments for share Dialog via

https://github.com/cds-snc/platform-forms-client/issues/1530#issuecomment-1431945267

Screen Shot 2023-02-16 at 3 26 30 PM

# Test instructions | Instructions pour tester la modification

- Share flow for both auth + non auth'd 
- Dialog on a smaller screen where scrolling is needed for the dialog

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:

- [ ] Have you completely tested the functionality of change introduced in this PR? Is the PR solving the problem it's meant to solve within the scope of the related issue?
- [ ] The PR does not introduce any new issues such as failed tests, console warnings or new bugs.
- [ ] If this PR adds a package have you ensured its licensed correctly and does not add additional security issues?
- [ ] Is the code clean, readable and maintainable? Is it easy to understand and comprehend.
- [ ] Does your code have adequate comprehensible comments? Do new functions have [docstrings](https://tsdoc.org/)?
- [ ] Have you modified the change log and updated any relevant documentation?
- [ ] Is there adequate test coverage? Both unit tests and end-to-end tests where applicable?
- [ ] If your PR is touching any UI is it accessible? Have you tested it with a screen reader? Have you tested it with automated testing tools such as axe?

### CODE REVIEW

**Comment by timarney** [comment]
@anikbrazeau flagging small update here:
https://github.com/cds-snc/platform-forms-client/pull/1636/files#diff-a6e486b021382fa7935492752ec9890787007f55c22fdc5c300ec95e11355a7bR265

File: public/static/locales/en/form-builder.json [hunk 1]
```diff
@@ -262,6 +262,7 @@
   "settingsResponseNotePublished": "To change the response delivery destination.",
   "settingsResponseTitle": "Response delivery destination",
```
Comment (anikbrazeau) [inline public/static/locales/en/form-builder.json:263]: Is this a person / email address?

File: public/static/locales/fr/form-builder.json [hunk 2]
```diff
@@ -262,6 +262,7 @@
   "settingsResponseNotePublished": "Pour modifier la destination de la livraison des réponses.",
   "settingsResponseTitle": "Adresse courriel pour la réception des réponses",
   "share": {
+    "formUser": "[FR] A GC Forms user",
     "copy": "Copier",
     "email": "Partager par courriel",
     "emailLabel": "Entrer l'adresse courriel",
```
Comment (anikbrazeau) [inline public/static/locales/fr/form-builder.json:268]:
```suggestion
  "settingsResponseNotePublished": "Pour modifier la destination de livraison des réponses.",
  "settingsResponseTitle": "Destination de livraison des réponses.",
  "share": {
    "formUser": "Un.e utilisateur.trice de Formulaires GC",
    "copy": "Copier",
    "email": "Partager par courriel",
    "emailLabel": "Entrer l'adresse courriel",
```

File: public/static/locales/en/form-builder.json (hunk 1 above)
Comment (timarney) [reply public/static/locales/en/form-builder.json:263]: In this case it's an email address

Screen Shot 2023-02-17 at 8 26 11 AM

File: public/static/locales/fr/form-builder.json [hunk 3]
```diff
@@ -259,9 +259,10 @@
   "settingsResponseHint1": "Fournissez une adresse électronique du gouvernement du Canada où envoyer les réponses aux questions du formulaire. Vous devez indiquer une adresse pour publier un formulaire.",
   "settingsResponseHint2": "Indiquer l’adresse courriel pour la réception des réponses",
   "settingsResponseNote": " Définissez les parametres de la livraison des réponses avant de tester la livraison et de publier.",
-  "settingsResponseNotePublished": "Pour modifier la destination de la livraison des réponses.",
-  "settingsResponseTitle": "Adresse courriel pour la réception des réponses",
+  "settingsResponseNotePublished": "Pour modifier la destination de livraison des réponses.",
+  "settingsResponseTitle": "Destination de livraison des réponses.",
   "share": {
+    "formUser": "Un.e utilisateur.trice de Formulaires GC",
```
Comment (anikbrazeau) [inline public/static/locales/fr/form-builder.json:265]:
```suggestion
    "formUser": "Un·e utilisateur·rice de Formulaires GC",
```

**Review by dsamojlenko — APPROVED** [review]

**Pull request merged**